docs/to_clarify.md

```diff
@@ -18,16 +18,26 @@
 
 1. Stated under the section "Possible Use Cases" that the solution should "begin their check-in journey before arrival and complete it upon arrival".
 
-    a. What needs to be done during the checking-in process?
+    a. What needs to be done during the digital checking-in process? Are there any particular details/ workflow where it must be adhered to provide a check-in process?
```

Comment:
Actually, I think this question can be removed. That's because we've already asked them above to 

> run through with us the current check-in/out process

Reply:
Are you referring to the forum question on 'pain points' or safe and secured? My intention was to cover missing requirements from their POV.

Reply:
The question on 'pain points'. @ahyong92 what do you think: is this question still needed?

Reply:
I don't think we should remove, it's something we have to keep in mind when they run through with us the process.

Reply:
I would have thought that "what needs to be done during the check in process" would have been covered as they explained about their check-in process :P 

But I'm ok either way. Just that Alvin's addition is slightly unclear and can be better reworded.

Reply:
Oh I get your point. @alvinlim540, I guess we should just remove part "a" from this point then?

Reply:
@Zhiyuan-Amos @ahyong92 I also think that my sentence is not specific enough. 
"a. What needs to be done during the digital checking-in process? Is there any of the hidden/back-end process must be adhered to provide a seamless and contact-less check-in process?"